### PR TITLE
#1587 Get dependent status excluding parents if both parents estranged or passed away

### PR DESCRIPTION
**Pull request by dheepak-aot**
## Modify the logic of dependent status in form.io

- [x] Change parent panel to be shown based off the current logic for determining if you are a dependent.
Created a new hidden variable `showParentInformation` which uses the current logic of determining  dependant status.
image

- [x] Change dependant_status value to be calculated based off the panel validation (point above) AND the logic from this question
 Form.io Javascript in VS Code
image

### CODE REVIEW

File: sources/packages/forms/src/form-definitions/sfaa2021-22.json [hunk 1]
```diff
@@ -31605,6 +31717,26 @@
...
+          "type": "hidden",
+          "tags": [],
+          "conditional": {
+            "show": "",
+            "when": null,
+            "eq": ""
+          },
+          "properties": {},
+          "calculateServer": true,
+          "calculateValue": "/**\r\n * When parent information is entered if student selects\r\n * both the parents passed away \r\n * or estranged from both of my parents then consider the\r\n * student as not dependant.\r\n */\r\nconst isDependant =\r\n  data.showParentInformation &&\r\n  (data.parentbcResident === \"yes\" || data.parentbcResident === \"no\");\r\n\r\nvalue = isDependant ? \"dependant\" : \"independant\";",
```
Comment (andrepestana-aot) [inline sources/packages/forms/src/form-definitions/sfaa2021-22.json:31738]: I think variables with the prefix `is` should be "boolean" accepting `true` and `false` or at most "yes" and "no".

Comment (dheepak-aot) [reply sources/packages/forms/src/form-definitions/sfaa2021-22.json:31738]: I didn't get the question.

**Review by andrepestana-aot — APPROVED** [review]
LGTM.

**Review by guru-aot — APPROVED** [review]
LGTM, nice work @dheepak-aot

**Review by andrewsignori-aot — APPROVED** [review]
Nice and precise work, looks good 👍

**Pull request merged**